experimenter/evaluate_pipeline_new.py

```diff
@@ -0,0 +1,282 @@
+import itertools as it
+import json
+import os
+
+from typing import Any, List, Tuple
+from uuid import UUID
+
+from d3m.metadata.pipeline import Pipeline
+from d3m import cli
+
+from experimenter.databases.d3m_mtl import D3MMtLDB
+from data_preparation_pipelines import K_FOLD_TABULAR_SPLIT_PIPELINE_ID, SCORING_PIPELINE_ID
+
+def save_pipeline_run_to_d3m_db(pipeline_run_path: str):
+    """ 
+    Saves a pipeline run document to the d3m database.
+
+    Parameters
+    ----------
+    pipeline_run_path : path_like str
+        path to pipeline_run document
+
+    Returns:
+    ----------
+    TODO
+
+    Raises:
+    ----------
+    TODO
+    """
+    d3m_db = D3MMtLDB()
+    return D3MMtLDB().save_pipeline_run(pipeline_run_path)
```

Comment:
@bjschoenfeld Does that mean this function is now not needed?

Reply:
See my last comment. We still need this for now.